### PR TITLE
zh:/docs/reference/config/type/_index.md

### PR DESCRIPTION
**Pull request by yanrongshi**
Updated:/zh/docs/reference/config/type/_index.md Chinese and added files

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/docs/reference/config/type/workload-selector/index.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
```
Comment (kebe7jun) [inline content/zh/docs/reference/config/type/workload-selector/index.html:1]: ？

File: content/zh/docs/reference/config/type/_index.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+title: 常见类型
```
Comment (kebe7jun) [inline content/zh/docs/reference/config/type/_index.md:2]: 最好能和实际内容一起提交，避免每次为了这种 _index.md 单独提一个 pr。

**Comment by Xunzhuo** [comment]
如果是 Work in Progress，记得提交 Draft PR，maintainer 就不会收到 notification，等 Ready 了再切换成正式 PR

File: content/zh/docs/reference/config/type/_index.md (hunk 2 above)
Comment (yanrongshi) [reply content/zh/docs/reference/config/type/_index.md:2]: 好的

**Pull request closed without merging**